cmd/mayactl/app/command/volumes_list.go

```diff
@@ -63,15 +63,15 @@ func (c *CmdVolumeOptions) RunVolumesList(cmd *cobra.Command) error {
 	}
 
 	out := make([]string, len(cvols.Items)+2)
-	out[0] = "Namespace|Name|Status|Type"
-	out[1] = "---------|----|------|----"
+	out[0] = "Namespace|Name|Status|Type|Capacity"
+	out[1] = "---------|----|------|----|--------"
```

Comment:
@pratik97 Thanks for contribution...
Can you please paste the output of `volume list` command which will show capacity of a openebs volume..that will be helpful.